integrations/huggingface-transformers/tutorials/text-classification/text-classification-cli.md

```diff
@@ -0,0 +1,583 @@
+  --per_device_train_batch_size 8 --per_device_eval_batch_size 32 --gradient_accumulation_steps 4 --preprocessing_num_workers 32 \
+  --seed 5114
+```
+
+## Sparse Transfer Learning with a Multi Input Custom Dataset (SICK)
+
+Beyond the built-in GLUE tasks, we can also use datasets for multiple input multi-class classification problems. The datasets can either be passed as Hugging Face hub model identifiers or via local CSV/JSON files.
+
+Let's try to transfer onto the [SICK (Sentences Involving Compositional Knowledge) Dataset](https://huggingface.co/datasets/sick),
+which includes 10,000 pairs of sentences with entailment relationships (`0=entailment, 1=neural, 2=contradiction`).
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation modifiers, we can turn them off by setting `--distill_teacher disable`.
+
+### Inspecting SICK Dataset
+
+Run the following to inspect the TweetEval
+
+```python
+from datasets import load_dataset
+from pprint import pprint
+
+sick = load_dataset("sick")
+print(sick)
+pprint(sick["train"][0])
+```
+
+We can see that each row dataset contains two input fields (`sentence_A` and `sentence_B`) which are strings representing the sequence pairs to be classified and a `label` field which is in `{0,1,2}` representing the entailment relationship.
+
+### Using a Hugging Face Dataset Identifier
+
+We can easily pass the `sick` Hugging Face dataset identifier to the script. Simply replace the `--task_name` argument with `--dataset_name sick --input_column_names 'sentence_A,sentence_B' --label_column_name label`.
+
+Run the following to kick off the training process:
+```bash
+sparseml.transformers.text_classification \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/text_classification/obert-base/pytorch/huggingface/mnli/pruned90_quant-none \
+  --distill_teacher disable \
+  --dataset_name sick --input_column_names 'sentence_A,sentence_B' --label_column_name 'label' \
+  --output_dir sparse_quantized_bert-text_classification_sick \
+  --do_train --do_eval --max_seq_length 128 --evaluation_strategy epoch --logging_steps 1000 --save_steps 1000 \
+  --per_device_train_batch_size 8 --per_device_eval_batch_size 32 --gradient_accumulation_steps 4 --preprocessing_num_workers 32 \
+  --seed 5114
+```
+
+You will notice that we used the same recipe as we did in the MNLI case (identified by the SparseZoo stub `zoo:nlp/text_classification/obert-base/pytorch/huggingface/mnli/pruned90_quant-none`). 
+
+Since the MNLI dataset is a multi sentence multi-class classification problem (similiarly, it is an entailment problem), we used the transfer learning recipe from the sentiment analysis task (a single sentence binary classification problem) as the starting point.
```

Comment:
Since the MNLI dataset is a multi-sentence and multi-class classification problem

Reply:
similiarly -> similarly